src/api/modelTypes.js

```diff
@@ -242,6 +242,7 @@ export type Stream = {|
   invite_only: boolean,
   is_announcement_only: boolean,
   history_public_to_subscribers: boolean,
+  stream_post_policy: number,
```

Comment:
This should be optional! It sure isn't always there.

Also should have a comment saying when it was introduced.

Reply:
Also should be next to `is_announcement_only`, as they're closely related.

Reply:
> Also should be next to is_announcement_only, as they're closely related.

I prefer alphabetically order, because when list is long it is easy to guess and locate desired item. You don't have to go through whole list.